_data/sites.json

```diff
@@ -10072,6 +10072,18 @@
             "meethue.com",
             "account.meethue.com"
         ]
+
+    },
+
+    {
+        "name": "phplist",
+        "url": "https://www.phplist.com/privacy",
+        "difficulty": "hard",
+        "notes": "Write an email to the email address in the link, requesting all data to be deleted.",
+        "notes_it": "Scrivere un’email all’indirizzo email indicato nella pagina del link, chiedendo che tutti i dati siano eliminati.",
```

Comment:
We can provide an email directly on the website, we would just need to do this:


```suggestion
        "notes": "Write an email requesting all data to be deleted.",
        "notes_it": "Scrivere un’email chiedendo che tutti i dati siano eliminati.",
        "email": "email@domain.com",
```

 (I can't see what the correct email is at the moment)

Reply:
the email is `info@phplist.com`
![Screen Shot 2022-09-25 at 11 41 38 AM](https://user-images.githubusercontent.com/42862428/192137395-95bafadd-c5bd-4f0f-b290-f6e4a8c4627a.jpg)